### PR TITLE
Fix syntax in p2-dsctl test-selector example.

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

File: bin/p2-dsctl/main.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ var (
 		This will output the hosts that match the selector,
 		The selector string uses same syntax as the kubernetes selectors without flags.
 		An example command is:
-		p2-dsctl test-selector "ice-cream in (the-fridge)"`,
+		p2-dsctl test-selector --selector "ice-cream in (the-fridge)"`,
```
Comment (rudle) [inline bin/p2-dsctl/main.go]: while you're here, would you mind making this more realistic? `hardware in (metal,lxc)` would be my preference

**Comment by rudle** [comment]
👍

File: bin/p2-dsctl/main.go [hunk 2]
```diff
@@ -81,7 +81,7 @@ var (
 		This will output the hosts that match the selector,
 		The selector string uses same syntax as the kubernetes selectors without flags.
 		An example command is:
-		p2-dsctl test-selector "ice-cream in (the-fridge)"`,
+		p2-dsctl test-selector --selector "hardware in (lxc,metal)"`,
```
Comment (bwester) [inline bin/p2-dsctl/main.go]: Shouldn't this be `sq/hardware`?

Comment (rudle) [inline bin/p2-dsctl/main.go]: depends who the example is for :( I could see this one going either way

We have helpful internal docs that describe valid selector types.

Comment (mpuncel) [inline bin/p2-dsctl/main.go]: i'll just put a placeholder instead

**Comment by rudle** [comment]
🐑 it

**Pull request merged**